### ISSUE TITLE
rule: blank-lines should ignore comments immediately before top level declarations

### ISSUE THREAD

**Issue by duaraghav8** [duaraghav8/Solium#6]
```js
contract Abc {
}

/*foo bar*/
contract Bcd {
}
```
This should pass without lint errors from the ```blank-lines``` rule because there's a 2-line gap between both contracts.
it should ignore the comment ```/*foo bar*/```

**Comment by 4gn3s** [comment]
This should include line comments also. I get a warning for this:
```
contract Abc {
function a() {}

//this function is temporary
function temp() {}
}
```

**Comment by duaraghav8** [comment]
@4gn3s yep agreed!
will notify you once this whole mess is sorted out

**Comment by 4gn3s** [comment]
I'm looking forward to it. The project is great, thanks!

**Comment by duaraghav8** [comment]
@4gn3s the bug has been fixed in v0.1.7 (latest).
Once you update, traverse to your root dir & run ```solium --sync```, you need not ```--init``` again. (sync adds entries for new rules added in the latest version into your ```.soliumrc.json```, I've added ```indentation```)

If you still see it malfunctioning, feel free to re-open the issue=)
